docker/docker.go

```diff
@@ -217,10 +218,10 @@ func IsContainerUpgradable(dClient *client.Client, imageCfg *container.Config, c
 	if err != nil {
 		return false, err
 	}
-	if containerInspect.Config.Image == imageCfg.Image {
-		logrus.Debugf("[%s] Container %s is not eligible for updgrade on host [%s]", plane, containerName, hostname)
-		return false, nil
+	if containerInspect.Config.Image != imageCfg.Image || !reflect.DeepEqual(containerInspect.Config.Cmd, imageCfg.Cmd) {
+		logrus.Debugf("[%s] Container %s is eligible for updgrade on host [%s]", plane, containerName, hostname)
```

Comment:
Nit: should %s be surrounded with [] for container? to kinda follow up the format you are using